packages/api/package.json

```
@@ -43,6 +43,7 @@
 		"build:types:esm": "rollup-type-bundler -d dist/esm -t .mts",
 		"build:types:cleanup": "tsx ../../scripts/clean-register-imports.mts",
 		"build:rename-cjs-register": "tsx ../../scripts/rename-cjs-register.mts",
+		"build:sync-mime-types": "tsx scripts/sync-mime-types.mts",
```

```suggestion
		"sync-mime-types": "tsx scripts/sync-mime-types.mts",
```

Since it's not a subscript of `build`